.readthedocs.yaml

```diff
@@ -1,26 +1,16 @@
-# .readthedocs.yaml
-# Read the Docs configuration file
-# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
-
-# Required
+# https://docs.readthedocs.io/en/stable/config-file/v2.html
 version: 2
-
-# Build documentation in the docs/ directory with Sphinx
+build:
+  os: ubuntu-20.04
+  tools:
+    python: mambaforge-4.10 # Python version is specified in Conda environment
 sphinx:
   configuration: docs/conf.py
-
-build:
-  image: latest
-
 python:
-  version: 3.8
```

Comment:
This information defining the correct Python version is missing; I've specified it in the environment file as recommended by the [RTD docs](https://docs.readthedocs.io/en/stable/config-file/v2.html#build-tools-python)